test/test-aritfacts/contract.cairo/contract_abi.json

```diff
@@ -0,0 +1,394 @@
+[
```

Comment:
How complicated would it be to not have the compilation artifacts here but have them compiled with the check.sh script in the plugin pat of the testing?